### DISCUSSION TITLE
SAI Peripheral support?

### DISCUSSION THREAD

**Discussion by herculanodavi** [modm-io/modm#776] [Ideas]
Hi there! 
I wonder if there are any plans on implementing the SAI peripheral that is available in most STMicro chips. 
I would be happy to try to implement it, too.

**Comment by chris-durand** [comment]
Hi @herculanodavi,

I am not aware of anyone implementing it, but there is an open [draft merge request](https://github.com/modm-io/modm/pull/769) for I²S audio playback support. We are always happy about contributions, so give it a try if you like. If there are any questions, feel free to ask.
